collections/nemo_nlp/nemo_nlp/utils/callbacks/sentence_classification.py

```diff
@@ -52,12 +48,12 @@ def eval_epochs_done_callback(global_vars, graph_fold):
     labels = np.asarray(global_vars['all_labels'])
     preds = np.asarray(global_vars['all_preds'])
     accuracy = sum(labels == preds) / labels.shape[0]
-    logger.info(f'Accuracy: {accuracy}')
+    nemo.logging.info(f'Accuracy: {accuracy}')
     i = 0
     if preds.shape[0] > 21:
         i = random.randint(0, preds.shape[0] - 21)
-    logger.info("Sampled preds: [%s]" % list2str(preds[i:i+20]))
-    logger.info("Sampled labels: [%s]" % list2str(labels[i:i+20]))
+    nemo.logging.info("Sampled preds: [%s]" % list2str(preds[i:i+20]))
```

Comment:
debug?

Reply:
Same argument as above:
I would prefer logging examples during evaluation. It helps to see if your model is training.
Therefore, I would argue to keep it at info